tests/utils.go

```diff
@@ -359,8 +359,9 @@ func EnsureKVMPresent() {
 			ready := true
 			// cluster is not ready until all nodes are ready.
 			for _, node := range nodeList.Items {
-				_, ok := node.Status.Allocatable[services.KvmDevice]
+				allocatable, ok := node.Status.Allocatable[services.KvmDevice]
 				ready = ready && ok
+				ready = ready && (allocatable.Value() > 0)
```

Comment:
If that only happens after our virt-handler-crash tests that makes sense. Might make sense to add it to an AfterEach of that test. If that happens also under other circumstances that looks like that fix would hide another issue.

Reply:
This code block covers the case when virt-handler is first brought up (e.g. `make cluster-sync`).

I wasn't aware we had tests that deliberately crashed virt-handler--in which case I agree that a similar approach will be needed before the test suite continues. or like you said, weird inconsistent behavior that's hard to diagnose could ensue in unrelated tests.